config/features.yml

```diff
@@ -2669,10 +2680,16 @@ features:
     actor_type: user
     description: If enabled, confirmation email will be sent with silent failure processing
   unique_user_metrics_logging:
-    # System level feature flag
-    actor_type:
+    actor_type: user
```

Comment:
Note: Changing this as we could reduce load by enabling for a percentage of users